internal/gatewayapi/testdata/backendtrafficpolicy-status-conditions-truncated.out.yaml

```diff
@@ -434,13 +450,13 @@ backendTrafficPolicies:
   kind: BackendTrafficPolicy
   metadata:
     creationTimestamp: null
-    name: target-httproute-with-attachment-conflict-truncated-ancestors
+    name: target-httproute-with-accepted-truncated-ancestors
```

Comment:
didn't follow alphabetical order when createtime is same?

Reply:
There is no sorting in this layer, it's following index order, the sorting has now moved to the provider which is ensuring index order

Reply:
Maybe I missed something, is there a test to make sure the sort worked as expected?